client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -160,11 +198,32 @@ class DomainDetailsForm extends Component {
 		} );
 	}
 
+	getAllFieldValues = () => {
+		const allFieldValues = Object.assign(
+			{},
+			formState.getAllFieldValues( this.state.form ),
+			this.state.registrantExtraInfo
```

Comment:
I've noticed that if you don't fill out, for example, your birth date or any extra info, it won't get submitted to the backend for validation at all:
<img width="534" alt="screen shot 2017-05-26 at 13 01 02" src="https://cloud.githubusercontent.com/assets/3392497/26492225/69c84a0e-4213-11e7-80d6-a729ce2afcac.png">
Is this expected?